src/module/scene/token-document/document.ts

```diff
@@ -295,8 +295,8 @@ class TokenDocumentPF2e<TActor extends ActorPF2e = ActorPF2e> extends TokenDocum
             huge: 3,
             grg: Math.max(token.width, 4),
         }[actor.size];
-        if (actor.isOfType("vehicle")) {
-            // Vehicles can have unequal dimensions
+        if (actor.isOfType("siegeWeapon") || actor.isOfType("vehicle")) {
```

Comment:
```suggestion
        if (actor.isOfType("siegeWeapon", vehicle")) {
```

Reply:
Great, no worries, I already had the hbs change pending locally. Fixed conflicts and incorporated your suggested clean up as well.